<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionBar</class>
 <widget class="QWidget" name="FunctionBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QToolButton" name="accountBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
  </widget>
  <widget class="QToolButton" name="transferBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QToolButton" name="contactBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>联系人</string>
   </property>
  </widget>
  <widget class="QToolButton" name="smartContractBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>182</y>
     <width>160</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>智能合约</string>
   </property>
  </widget>
  <widget class="QToolButton" name="feedBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>160</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>喂价</string>
   </property>
  </widget>
  <widget class="QToolButton" name="multiSigBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>276</y>
     <width>160</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>多重签名</string>
   </property>
  </widget>
  <widget class="QToolButton" name="advanceBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>高级</string>
   </property>
  </widget>
  <widget class="QToolButton" name="accountListBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>160</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>账户列表</string>
   </property>
  </widget>
  <widget class="QToolButton" name="billInfoBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>228</y>
     <width>160</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>账单信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>516</y>
     <width>26</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>536</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(83,90,109);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentChoiceLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>140</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="assetComboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{background-color:transparent;border:none;color: rgb(192, 196, 212);padding-left: 5px;}
QComboBox::drop-down {background:transparent;border:none;}
QComboBox::down-arrow {image: url(:/ui/wallet_ui/downArrow.png);}
QComboBox::down-arrow:on {image: url(:/ui/wallet_ui/downArrow_on.png);}
QComboBox QAbstractItemView {outline: 0px;}
QComboBox QAbstractItemView::item { min-height: 30px;color:rgb(192,196,212);selection-color: white;background-color: rgb(48,52,72);selection-background-color: rgb(44,48,67);}
QComboBox QAbstractItemView::item:hover {color: white;background-color: rgb(44,48,67);}
</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
   <item>
    <property name="text">
     <string>ETH</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="totalBalanceLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 UB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="scanLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>40</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(192, 196, 212);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Total</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(24,250,239);</string>
   </property>
   <property name="text">
    <string>0 UB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>140</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(192, 196, 212);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Total</string>
   </property>
  </widget>
  <widget class="QToolButton" name="choiceBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
